packages/conventional-changelog-core/lib/merge-config.js

```diff
@@ -251,6 +251,16 @@ function mergeConfig (options, context, gitRawCommitsOpts, parserOpts, writerOpt
         gitRawCommitsOpts.reverse = gitRawCommitsOpts.reverse || true
       }
 
+      if (options.mergeCommitFilter) {
+        if (options.mergeCommitFilter === 'exclude') {
+          gitRawCommitsOpts.merges = false
+        } else if (options.mergeCommitFilter === 'only-merges') {
+          gitRawCommitsOpts.merges = true
+        } else if (options.mergeCommitFilter === 'include') {
+          // noop; just not set `merges` option in rawCommitOpts
```

Comment:
If this is a no-op then there is no need for `if`.

Reply:
Have a look at https://github.com/conventional-changelog/conventional-changelog/commit/ac1f189550e1acc92f77e1b4fdcbd8581c10b317